app/javascript/components/molecules/StatusItems.js

```diff
@@ -1,46 +1,9 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import moment from 'moment'
 
-import StatusPill from '../atoms/StatusPill'
+import StatusItem from './StatusItem'
 
 import StatusItemShape from '../../utils/shapes/StatusItemShape'
-import { getSubStatusLabel } from '../../utils/statusUtils'
-
-const StatusDate = ({ timeStamp }) => (
-  <div className='status-item-date'>
-    {moment.unix(timeStamp).format('MMMM Do YYYY')}
-  </div>
-)
-
-StatusDate.propTypes = { timeStamp: PropTypes.number }
-
-const StatusItem = ({ statusItem }) => {
```

Comment:
One issue with shallow rendering is if we declare multiple components in a single file and only export one of them, the shallow rendering won't actually be testing the child components. If you want the children to be tested, you should either move the child into its own file or add an `export` to the child component definition that's only used by the spec files.